src/resource/solidDataset.ts

```diff
@@ -217,18 +222,21 @@ export async function saveSolidDatasetAt<Dataset extends SolidDataset>(
     ...options,
   };
 
-  const requestInit = isUpdate(solidDataset, url)
-    ? await prepareSolidDatasetUpdate(solidDataset)
-    : await prepareSolidDatasetCreation(solidDataset);
+  // const datasetWithChangelog = withChangeLog(solidDataset);
```

Comment:
```suggestion
```

Reply:
Oops, I commented that out to make sure my new test was failing without my fix, and then re-implemented my fix on the next line, because why not ^^.
5ad3ec10eda88b7fa7a735c71cb127706140e7bb